LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -93,28 +116,42 @@ List<Module> GetModuleList()
 
         public void AssignModulesToStudents()
         {
+            string action = "Assing Modules to students";
+            int count = 0;
+            ReportHeader(actionName: action);
             foreach (var student in studentList)
             {
                 student.CompletedModuleList = GetModuleList();
                 Console.WriteLine(student);
             }
+
+            ReportSummary(actionName: action, count: count);
```

Comment:
count is always 0, at line 120 you have to assign it: count = studentList.Count to be able to use it later.

Reply:
This is not correct either, now you always print the total count. The total count is required in the summary, but in ReportItem() you should print the current index.